metrix-mapping/src/main/java/com/powsybl/metrix/mapping/log/ScalingDownLimitChangeSynthesis.java

```diff
@@ -0,0 +1,59 @@
+package com.powsybl.metrix.mapping.log;
+
+public class ScalingDownLimitChangeSynthesis extends AbstractLogBuilder implements LogDescriptionBuilder {
+
+    private String timeSeriesName;
+    private String violatedVariable;
+    private String variable;
+    private String evolution;
+    private String type = "";
+
+    public ScalingDownLimitChangeSynthesis violatedVariable(String violatedVariable) {
+        this.violatedVariable = violatedVariable;
+        return this;
+    }
+
+    public ScalingDownLimitChangeSynthesis variable(String variable) {
+        this.variable = variable;
+        return this;
+    }
+
+    public ScalingDownLimitChangeSynthesis timeSeriesName(String timeSeriesName) {
+        this.timeSeriesName = timeSeriesName;
+        return this;
+    }
+
+    public ScalingDownLimitChangeSynthesis min() {
+        this.evolution = " decreased";
+        return this;
+    }
+
+    public ScalingDownLimitChangeSynthesis max() {
+        this.evolution = " increased";
+        return this;
+    }
+
+    public ScalingDownLimitChangeSynthesis mapped() {
+        this.type = "mapped";
+        return this;
+    }
+
+    public ScalingDownLimitChangeSynthesis baseCase() {
+        this.type = "base case";
+        return this;
+    }
+
+    public ScalingDownLimitChangeSynthesis buildLimitChange() {
+        this.label = SCALING_DOWN_PROBLEM + "at least one " + violatedVariable + evolution + TS_SYNTHESIS;
```

Comment:
QCA : RMQ => what do you think of String.format( "{a: %s, b: %s, c: %s}", a, b, c ); to avoid this type of concatenation